process/track/argBlockTrack.go

```diff
@@ -19,7 +19,7 @@ type ArgBaseTracker struct {
 	RoundHandler     process.RoundHandler
 	ShardCoordinator sharding.Coordinator
 	Store            dataRetriever.StorageService
-	StartHeaders     map[uint32]data.HeaderHandler
+	StartHeader      data.HeaderHandler
```

Comment:
leave shardIDs and maps everywhere - as it is easier to make common code with elrond-go - we will just input 0 everywhere. Or make another implementation which will call these with 0.